tests/functional/api/test_statistics.py

```diff
@@ -0,0 +1,37 @@
+"""
+GitLab API: https://docs.gitlab.com/ee/api/statistics.html
+"""
+
+
+from tests.functional.conftest import reset_gitlab
```

Comment:
fixtures doesn't have to be imported. Just add the fixture to your function arguments and it will be executed.

Reply:
`reset_gitlab` isn't a fixture as of now - it's just a helper function as far as I can see. Not sure if we'd want to convert it into one as a function call is made to it within a different fixture.

Reply:
Agreed that this is a helper now, we could potentially move it to `tests.functional.helpers` later to avoid this confusion. Not needed for this PR though